### PR TITLE
Hocs 2527 additional config case creator

### PR DESCRIPTION
**Pull request by philbarton**
Add missing config to get the case creator working in docker.

Also :
- Fixed postgres error
- Added a sleep, to aws_cli. Without the sleep, it gets run multiple times.
- Set images to main rather than latest

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -159,8 +162,7 @@ services:
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:8080/actuator/health"]
       start_period: 240s
-    image: quay.io/ukhomeofficedigital/hocs-casework
-    #image: hocs-casework-test
+    image: quay.io/ukhomeofficedigital/hocs-casework:${CASEWORK_TAG-branch-main}
```
Comment (LiamMacP) [inline docker/docker-compose.yml:165]: Does the following need the colon like the fronend above?

**Comment by zuzak** [comment]
Once you've got to the point that this is ready to be merged, close this PR and reopen it against [ukhomeoffice/hocs](https://github.com/UKHomeOffice/hocs) instead.

I'm not sure what the future of hocs-frontend's docker-compose should be; maybe we should just remove it?

File: docker/docker-compose.yml (hunk 1 above)
Comment (philbarton) [reply docker/docker-compose.yml]: It works with both syntax's!! I've change them all to ${X_TAG:-branch-main} for consistancy

**Review by robertdeniszczyc2 — APPROVED** [review]
Looks good to me

**Pull request merged**